test/values_test.js

```diff
@@ -120,4 +121,37 @@ describe('Values', function() {
     assert.equal(json.toJSON(test_query), test_query_json);
     assert.deepEqual(json.parseJSON(test_query_json), test_query);
   });
+
+  it('pretty print', function() {
+    var assertPrint = function(value, expected) {
+      assert.equal(expected, util.inspect(value, {depth: null}));
+      assert.equal(expected, value.toString());
+    };
+
+    assertPrint(new Ref('cls', values.Native.CLASSES), 'Class("cls")');
+    assertPrint(new Ref('db', values.Native.DATABASES), 'Database("db")');
+    assertPrint(new Ref('idx', values.Native.INDEXES), 'Index("idx")');
+    assertPrint(new Ref('fn', values.Native.FUNCTIONS), 'Function("fn")');
+    assertPrint(new Ref('key', values.Native.KEYS), 'Ref(Keys(), "key")');
+
+    assertPrint(values.Native.CLASSES, 'Classes()');
+    assertPrint(values.Native.DATABASES, 'Databases()');
+    assertPrint(values.Native.INDEXES, 'Indexes()');
+    assertPrint(values.Native.FUNCTIONS, 'Functions()');
+    assertPrint(values.Native.KEYS, 'Keys()');
+
+    var db = new Ref("db", values.Native.DATABASES);
+    assertPrint(new Ref('cls', values.Native.CLASSES, db), 'Class("cls", Database("db"))');
+    assertPrint(new Ref('db', values.Native.DATABASES, db), 'Database("db", Database("db"))');
+    assertPrint(new Ref('idx', values.Native.INDEXES, db), 'Index("idx", Database("db"))');
+    assertPrint(new Ref('fn', values.Native.FUNCTIONS, db), 'Function("fn", Database("db"))');
+    assertPrint(new Ref('key', values.Native.KEYS, db), 'Ref(Keys(Database("db")), "key")');
+
+    assertPrint(new FaunaTime('1970-01-01T00:00:00.123456789Z'), 'Date("1970-01-01T00:00:00.123456789Z")');
+    assertPrint(new FaunaDate('1970-01-01'), 'Date("1970-01-01")');
+
+    assertPrint(new SetRef({'match': new Ref('idx', values.Native.INDEXES)}), 'SetRef({ match: Index("idx") })');
+    assertPrint(new Bytes('1234'), 'Bytes("1234")');
+    assertPrint(new Query({'lambda': 'x', 'expr': {'var': 'x'}}), 'Query({ lambda: \'x\', expr: { var: \'x\' } })');
```

Comment:
Is it worth making this look like a function?

Reply:
What do you mean? the lambda inside `Query(...)`? So, since @freels said he didn't want to see `[object Object]` in our doc responses, I just stringify the internal value.

Reply:
I though it'd be nice to see lambda as a JS function. Like `Query(x => x)`, if possible.

Reply:
If not that, then `Lambda` constructor syntax.

Reply:
The `Lambda` syntax could later be copy pasted into a shell. So it can help people reading the examples

Reply:
Well, that was not the best code ever, but do the job